airflow/contrib/hooks/kafka_consumer_hook.py

```diff
@@ -0,0 +1,71 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from airflow.hooks import base_hook as BaseHook
+from kafka import KafkaConsumer
+
+
+class KafkaConsumerHook(BaseHook):
+
+    DEFAULT_HOST = 'localhost'
+    DEFAULT_PORT = 9092
+
+    def __init__(self, conn_id, topic):
+        super(KafkaConsumerHook, self).__init__(None)
+        self.conn = self.get_connection(conn_id)
```

Comment:
Let's move this call to get_connection, example: https://github.com/apache/airflow/commit/4bec1cc489f5d19daf7450c75c3e8057c9709dbd

Reply:
This is to avoid DB call on Initialisation.

Reply:
Address in newest commit.